contracts/anchors/bridged/AnchorPoseidon2.sol

```diff
@@ -92,8 +94,9 @@ abstract contract AnchorPoseidon2 is MerkleTreePoseidon, ReentrancyGuard {
     uint32 insertedIndex = _insert(_commitment);
     commitments[_commitment] = true;
     _processDeposit();
+    updateNonce++;
```

Comment:
This isn't correct as per what we discussed. The update nonce is the # of times the anchors edges have been updated. This would be a `depositNonce`?

Reply:
I don't think we ever came to an agreement about the nonce scheme. So I went ahead an implemented what I think the best implementation would be. I don't think an updateNonce in the way you talk about it makes sense as really in the chainbridge scheme a nonce is generated upon deposit and used throughout on the voting proposals on the other chain after. Therefore we can change the name to depositNonce for this global variable.   The way i see it working is:
1) deposit nonce incremented and emitted upon deposit into anchor
2) relayers vote on this nonce along with the other data emmited after deposit (merkleroot and index)

this is similar to chainbridge, but the nonce is generated in the anchor deposit instead of the bridge deposit which we dont have.